pageserver/src/deletion_queue/deleter.rs

```diff
@@ -88,7 +101,10 @@ impl Deleter {
                     self.accumulator.clear();
                 }
                 Err(e) => {
-                    warn!("DeleteObjects request failed: {e:#}, will retry");
+                    if self.cancel.is_cancelled() {
+                        return Err(DeletionQueueError::ShuttingDown);
+                    }
+                    warn!("DeleteObjects request failed: {e:#}, will continue trying");
```

Comment:
Should we allow this globally for all tests?

Reply:
Probably not, now that it is behind a 10x retry it's quite unlikely to bubble up  to this level.